packaging/makeself/install-or-update.sh

```diff
@@ -248,6 +248,24 @@ fi
 
 # -----------------------------------------------------------------------------
 
+echo "Configure TLS certificate paths"
+if [ ! -L /opt/netdata/etc/ssl ] && [ -d /opt/netdata/etc/ssl ] ; then
+  echo "Preserving existing user configuration for TLS"
+else
+  if [ -d /etc/pki/tls ] ; then
+    echo "Using /etc/pki/tls for TLS configuration and certificates"
+    ln -sf /etc/pki/tls /opt/netdata/etc/ssl
+  elif [ -d /etc/ssl ] ; then
+    echo "Using /etc/ssl for TLS configuration and certificates"
+    ln -sf /etc/ssl /opt/netdata/etc/ssl
+  else
+    echo "Using bundled TLS configuration and certificates"
+    ln -sf /opt/netdata/share/ssl /opt/netdata/etc/ssl
```

Comment:
I am still not convinced we should be symlinking anything from the host system here. Someone explain to me again one more time with very real tangile use-cases why we'd care about the host's CA certs?

Reply:
Standard SSL/TLS intercepting captive portals/transparent proxies on enterprise networks.

If we're running in such a network, connection to Netdata Cloud will fail if we don't have the local CA for the intercepting system. Preferentially using the system CA bundle means that admins can just go through the normal setup for such a scenario (namely, inject their own CA into the system trust store) and Netdata will just work.

Reply:
Okay I actually buy that argument. Thanks!